### DISCUSSION TITLE
已购买专业版，邮箱无法收到验证码

### DISCUSSION THREAD

**Discussion by Vindalfr111** [getlantern/lantern-forum-cn#950] [General]
![Screenshot_2024-10-10-17-43-03-794_com eg android AlipayGphone](https://github.com/user-attachments/assets/0b5270c1-5634-42fd-903c-e7285cf4b569)
邮箱236950425@qq.com

**Comment by YongboShen** [comment]
同样的问题，到底还解不解决啊？能不能给个话？

**Reply by MaxMKMax** [reply]
@YongboShen 您好. 请您提供我支付单据和账号邮箱. 这样我才能帮您提交问题上去

**Comment by MaxMKMax** [comment]
@Vindalfr111 您好. 我已经帮您提交问题上去. 估计几天内会帮您处理好. 谢谢等待.

**Reply by Vindalfr111** [reply]
问题反映两周了，邮箱还是收不到验证码

**Comment by silverAnswerer** [comment]
@Mike777Khoo   你好   这边也是相同的问题 购买付费之后邮箱收不到授权邮件，然后付费完成后直接就退出了

**Comment by YongboShen** [comment]
您好，这是我的支付单据，邮箱是 ***@***.***


Mike K ***@***.***> 于 2024年10月11日周五 下午5:39写道：

**Comment by MaxMKMax** [comment]
@Vindalfr111 
您好. 这是您的激活码:
XXGYY-Y9XXH-GGWKD-YB8WX-BP2R9
